components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java

```diff
@@ -54,6 +54,7 @@
 
     private static Log log = LogFactory.getLog(SiddhiTryItClient.class);
     private static Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
+    private static SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
```

Comment:
@thiliA 
Shall we make this as an instance variable since SimpleDateFormat is not threadsafe [1,2]? Since this is being called from the ajax call and the ajax call creates a SiddhiTryItClient per call, making it as an instance variable should ensure that different threads have its own copy of SimpleDateFormat.

[1] http://javarevisited.blogspot.com/2012/03/simpledateformat-in-java-is-not-thread.html
[2] http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html